temporal-opentracing/src/main/java/io/temporal/opentracing/provider/DataDogOpenTracingSpanBuilderProvider.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing.provider;
+
+import io.temporal.opentracing.SpanBuilderProvider;
+import io.temporal.opentracing.SpanCreationContext;
+import io.temporal.opentracing.internal.SpanBuilderFromSpanContentProvider;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This implementation of {@link SpanBuilderProvider} names and tags the OpenTracing spans in a way
+ * that is compatible with DataDog's APM tooling.
+ */
+public class DataDogOpenTracingSpanBuilderProvider extends SpanBuilderFromSpanContentProvider
+    implements SpanBuilderProvider {
+
+  /**
+   * Uses just the operation type as the name, e.g. "StartActivity"
+   *
+   * @param context
+   * @return
```

Comment:
Please provide short descriptions for parameters and the return value or delete the "@param context" and "@return", it's meaningless now.